docs/4.x/tests-github.md

```diff
@@ -0,0 +1,50 @@
+---
+category: Develop
+previous: tests-js-tracker
```

Comment:
If we define a previous/next here, then we also need up date the previous and next on these two pages `tests-js-tracker` and `tests-troubleshooting` as otherwise the order won't work anymore. 

Like in `tests-js-tracker.md` we'd expect a `next: tests-github.md` for example